test/index.test.js

```diff
@@ -17,3 +17,20 @@ describe('Homepage', () => {
       });
   });
 });
+
```

Comment:
Nice work here. Same feedback as the previous PR. Pls group your tests into files related to the tests being written (e.g party.test.js)

Reply:
I'll work on it